### PR TITLE
🧪 Add client-side experiments config file

### PR DESCRIPTION
**Pull request by danielrozenberg**
Related to #34013

### CODE REVIEW

File: build-system/global-configs/README.md [hunk 1]
```diff
@@ -42,11 +42,41 @@ If `build-system/global-configs/custom-config.json` exists at build time, its pr
...
+```json
 {
   "canary": 1,
   "chunked-amp": 1,
   "version-locking": 0,
   "cdnUrl": "https://example.com/amp"
 }
 ```
+
+# client-side-experiments-config.json
```
Comment (rsimha) [inline build-system/global-configs/README.md:54]: Now that this file has detailed explanations for `custom-config.json` and `client-side-experiments-config.json`, the top `Flags` section looks a little weird on its own.

Can we change its title from `Flags` to something like `{prod|canary}-config.json`, and add a sentence explaining the purpose of the two files (before the _"Please be aware that canary-config.json..."_ line)?

Okay to do in a follow up if you prefer.

File: build-system/global-configs/README.md [hunk 2]
```diff
@@ -42,11 +42,41 @@ If `build-system/global-configs/custom-config.json` exists at build time, its pr
...
+    {
+      "name": "version-locking",
+      "percentage": 1,
+      "rtvPrefix": ["01", "02", "03", "04", "05"]
+    }
+  ]
+}
+```
+
+Once merged onto the `main` branch, this file is automatically picked up by the AMP CDN (usually within 1-2 hours from PR merge) and its content is injected into the v0.\[m\]js file. AMP caches can also inject the contents of this file verbatim into pages inside a `<script language=text/json id=__AMP_EXP>{...}</script>` element.
```
Comment (rsimha) [inline build-system/global-configs/README.md:82]: optional formatting nit for readability:
```suggestion
Once merged onto the `main` branch, this file is automatically picked up by the AMP CDN (usually within 1-2 hours from PR merge) and its content is injected into `v0.\[m\]js`. AMP caches can also inject the contents of this file verbatim into pages inside a `<script language=text/json id=__AMP_EXP>{...}</script>` element.
```

**Review by rsimha — APPROVED** [review]
LGTM 👍

File: build-system/global-configs/README.md (hunk 1 above)
Comment (danielrozenberg) [reply build-system/global-configs/README.md:54]: Done, PTAL

Comment (rsimha) [reply build-system/global-configs/README.md:51]: Perfect. Thank you!

File: build-system/global-configs/README.md [hunk 3]
```diff
@@ -42,11 +39,41 @@ If `build-system/global-configs/custom-config.json` exists at build time, its pr
...
+    {
+      "name": "version-locking",
+      "percentage": 1,
+      "rtvPrefix": ["01", "02", "03", "04", "05"]
+    }
+  ]
+}
+```
+
+Once merged onto the `main` branch, this file is automatically picked up by the AMP CDN (usually within 1-2 hours from PR merge) and its content is injected into `v0.\[m\]js`. AMP caches can also inject the contents of this file verbatim into pages inside a `<script language=text/json id=__AMP_EXP>{...}</script>` element.
```
Comment (rsimha) [inline build-system/global-configs/README.md:79]: My bad, now that the filename has backticks, it no longer needs backslashes 🤦 
```suggestion
Once merged onto the `main` branch, this file is automatically picked up by the AMP CDN (usually within 1-2 hours from PR merge) and its content is injected into `v0.[m]js`. AMP caches can also inject the contents of this file verbatim into pages inside a `<script language=text/json id=__AMP_EXP>{...}</script>` element.
```

**Pull request merged**